The `network` module tries to pick a default network backend based on the
availability of executables used by those backends, but Void removes those that
depend on systemd. This patch preserves the "use a given backend if it's
included in the image" because modules that aren't installed should never be
included, but replaces the default with network-legacy.

--- a/modules.d/40network/module-setup.sh
+++ b/modules.d/40network/module-setup.sh
@@ -17,13 +17,7 @@
         done;
 
         if [ -z "$network_handler" ]; then
-            if find_binary wicked  &>/dev/null ; then
-                network_handler="network-wicked"
-            elif [[ -x $dracutsysrootdir/usr/libexec/nm-initrd-generator ]]; then
-                network_handler="network-manager"
-            else
-                network_handler="network-legacy"
-            fi
+            network_handler="network-legacy"
         fi
     echo "kernel-network-modules $network_handler"
     return 0
